bytecode/cfg.py

```diff
@@ -141,7 +141,11 @@ def update_size(pre_delta, post_delta, size, maxsize):
         # jump is taken.
         if instr.has_jump():
             taken_size, maxsize = update_size(
```

Comment:
I would do the following to not have to pass check_pre_and_post to the instruction.
```python
effects = instr.pre_and_post_stack_effect(jump=True) if check_pre_and_post else (0, instr.stack_effect(jump=True)
taken_size, maxsize = update_size(*effects, size, maxsize)
```
and something similar for the jump False case.

Reply:
yea, that was also my first thought, but it seems that the coverage test would fail, that's why I changed it to this way